### PR TITLE
Interactive K-RVEA

### PR DESCRIPTION
**Pull request by ppouyaa**
I have added the model management in the utilities folder and added a notebook example

### CODE REVIEW

File: desdeo_tools/utilities/model_management.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+from numpy.core.numeric import indices
+from numpy.lib.arraysetops import unique
+from desdeo_tools.scalarization.ASF import SimpleASF
+from numba import njit
+import numpy as np
+from desdeo_problem.problem.Objective import ScalarDataObjective
+import pandas as pd
+import copy
+from desdeo_problem.surrogatemodels.SurrogateModels import GaussianProcessRegressor
```
Comment (light-weaver) [inline desdeo_tools/utilities/model_management.py:9]: desdeo-tools should not depend on methods from desdeo-problem. If you need to use code from both repositories, implement these algorithms in desdeo-emo.

File: desdeo_tools/utilities/model_management.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        if indicies.size == 0:
+            indicies = None
+    if indicies is None:
+        return None
+    else:
+
+        return indicies
+
+
+def ikrvea_mm(reference_point, evolver , problem, u: int) -> float:
```
Comment (light-weaver) [inline desdeo_tools/utilities/model_management.py:40]: Do you need the evolver at all? You never use the EA methods. Instead, you only use the objective/decision/uncertainity vectors.

File: desdeo_tools/utilities/model_management.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+from desdeo_problem.problem.Objective import ScalarDataObjective
+import pandas as pd
+import copy
+from desdeo_problem.surrogatemodels.SurrogateModels import GaussianProcessRegressor
+from sklearn.gaussian_process.kernels import DotProduct,\
+    WhiteKernel, RBF, Matern, ConstantKernel
+
+
+
+def remove_duplicate(X, archive_x):
```
Comment (light-weaver) [inline desdeo_tools/utilities/model_management.py:15]: What's X and archive_x?

File: desdeo_tools/utilities/model_management.py [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+            indicies = None
+    if indicies is None:
+        return None
+    else:
+
+        return indicies
+
+
+def ikrvea_mm(reference_point, evolver , problem, u: int) -> float:
+    """ Computes the additive epsilon-indicator between reference front and current approximating front.
```
Comment (light-weaver) [inline desdeo_tools/utilities/model_management.py:41]: Outdated docstring

File: desdeo_tools/utilities/model_management.py [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+        non_duplicate_dv = evolver.population.individuals[nd]
+        non_duplicate_obj = evolver.population.objectives[nd]
+        non_duplicate_unc = evolver.population.uncertainity[nd]
+    else:
+        non_duplicate_dv = evolver.population.individuals
+        non_duplicate_obj = evolver.population.objectives
+        non_duplicate_unc = evolver.population.uncertainity
+
+
+    asf_solutions = SimpleASF([1,1,1]).__call__(non_duplicate_obj, reference_point)
```
Comment (light-weaver) [inline desdeo_tools/utilities/model_management.py:70]: This only works for 3D?

**Review by light-weaver — CHANGES_REQUESTED** [review]

**Comment by ppouyaa** [comment]
I will move it to desdeo_emo

**Pull request closed without merging**